client/src/components/ReviewTile.js

```diff
@@ -1,11 +1,48 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
 
 const ReviewTile = (props) => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const deleteReview = async () => {
+    try {
+      const response = await fetch(`/api/v1/reviews/${props.id}`, {
+        method: "delete",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      }
+      const respBody = await response.json()
+      setShouldRedirect(true)
```

Comment:
We don't want to redirect and have to refetch all data, but update the state of `reviews` to remove the one that was deleted 
- we would probably need this `delete` request to be defined in the same component where the state for all `reviews` are
- we need to remove/filter out the deleted review from the reviews in state, to do this we need to make sure we use non-mutating methods (like filter) that will return a new array for us to set state